### ISSUE TITLE
Registries in ISO/IEC

### ISSUE THREAD

**Issue by murata2makoto** [w3c/epub-specs#999]
EPUB 3.2 is expected to rely on registries in IDPF or W3C.  Furthermore, EPUB accessibility relies on schema.org, which is an external registry.

In ISO or IEC, the use of registries is not easy.   We do not have to be intimidated, but procedural issues around registries will take much more effort than you can possibly imagine.   In particular, since schema.org is not under our control, the use of schema.org in EPUB accessibility will be much more difficult than the use of IDPF or W3C registries in EPUB 3.2.

Here are some pointers to relevant information.

### Consolidated JTC 1 Supplement 2017

Available at:
http://isotc.iso.org/livelink/livelink?func=ll&objId=9482942&objAction=browse&viewType=1

See 
- Subclause 2.12 Registration authorities
- Annex H (normative) Registration authorities

### JTC1 Standing Document 16: Registration Authorities

Available in the zip file shown as the first bullet in the web page at:

https://isotc.iso.org/livelink/livelink?func=ll&objId=9384365&objAction=browse&sort=name

### A precedent in SC34

SC34 developed an ISO/IEC standard for the procedures around a glyph registry.

[ISO/IEC 10036:1996 ed.2-
Information technology -- Font information interchange -- Procedures for registration of font-related identifiers
](https://www.iso.org/standard/24744.html)

This standard has only 17 pages.  We will have to do something similar for registries.

**Comment by TzviyaSiegman** [comment]
@murata2makoto The whole point of using schema.org is that it is a Web standard. We are trying to integrate with the Web. This presents a very big problem.

**Comment by murata2makoto** [comment]
I agree that schema.org is becoming a web standard.  But it is not a standard in the ISO or IEC world.

**Comment by iherman** [comment]
Unfortunately, @murata2makoto is right; W3C also had to introduce a special agreement to allow references to schema.org from a W3C standard.

Is there a means at ISO to ask for special dispensation of some sort? I do not think we should move away from schema.org

**Comment by murata2makoto** [comment]
In the case of EPUB A11Y, I think that the most practical solution is to drop all references to schema.org from it and then introduce them to EPUB A11Y Techniques.

> Is there a means at ISO to ask for special dispensation of some sort?

This is extremely difficult.  There are many other issues around the way ISO/IEC standards are edited, but I am not aware of much progress.

**Comment by mattgarrish** [comment]
> I think that the most practical solution is to drop all references to schema.org from it and then introduce them to EPUB A11Y Techniques.

We just need to make sure that we don't introduce too much ambiguity in doing this. It's easy to abstract the requirements ("must describe the accessibility features" instead of "must include schema:accessibilityFeature"), but we need to consider the wording carefully so that someone can't just write a long dc:description and claim they are conforming.

We need to craft some wording that makes clear that each item needs to be represented by a unique metadata item, in other words. This would make schema.org the only practical technique at this time.

**Comment by iherman** [comment]
> the most practical solution is to drop all references to schema.org from it and then introduce them to EPUB A11Y Techniques.

That would be most unfortunate, to say the least. Many in this community were active in adding some terms to schema.org, with the acknowledgement that schema.org has become fairly essential on the Web. Removing that from EPUB A11Y would mean a counterintuitive step, when I believe everyone agrees that having good Web search results is essential for the long term business model of that community...

**Comment by mattgarrish** [comment]
> That would be most unfortunate, to say the least.

It's not ideal, but it's also not that different from what we had to do to remain technology agnostic when adding the metadata as a reporting recommendation (it also doesn't call out schema.org).

ONIX also (natively) allows some of the same metadata to be expressed, so in abstracting we might also better harmonize the requirement in section 6 to include accessibility metadata in distribution.

**Comment by mattgarrish** [comment]
> when adding the metadata as a reporting recommendation

"in WCAG" is the missing context here.

**Comment by mattgarrish** [comment]
This change was made for the ISO document so it's already incorporated into the new draft text.
